guides/v2.2/cloud/project/project-conf-files_services.md

```diff
@@ -6,96 +6,139 @@ functional_areas:
+This name value is used in the `relationships` property of the `.magento.app.yaml` configuration file:
 
-Identifies the service in the project. The `name` can consist only of lower case alphanumeric characters: `a` to `z` and `0` to `9`. For example, Redis is entered as `redis`.
+```yaml
+relationships:
+    redis: "<name>:redis"
+```
 
-You can have multiple instances of each service type. For example, you could have multiple Redis instances. For example, we use multiple Redis instances, one for session and one for cache.
+You can name multiple instances of each service type. For example, we could use multiple Redis instances—one for session and one for cache.
 
 ```yaml
 redis:
-    type: redis:3.0
+    type: redis:<version>
 
 redis2:
-    type: redis:3.0
+    type: redis:<version>
 ```
 
-Be aware, if you rename a service in the `services.yaml` file, the following is **permanently removed**:
+Renaming a service in the `services.yaml` file **permanently removes** the following:
 
 -  The existing service before creating a new service with the new name you specify.
 -  All existing data for the service is removed. We strongly recommend you [snapshot your environment]({{ page.baseurl }}/cloud/project/project-webint-snap.html) before you change the name of an existing service.
 
-### `type` {#cloud-yaml-services-type}
+### `type`
 
-The `type` of your service in the format: `type:version`
+The `type` value specifies the version of the service. For example:
 
-We support and deploy the following services for you:
+```yaml
+mysql:
+    type: mysql:10.2
+```
 
--  [`mysql`]({{ page.baseurl }}/cloud/project/project-conf-files_services-mysql.html)
--  [`redis`]({{ page.baseurl }}/cloud/project/project-conf-files_services-redis.html)
--  [`elasticsearch`]({{ page.baseurl }}/cloud/project/project-conf-files_services-elastic.html)
--  [`rabbitmq`]({{ page.baseurl }}/cloud/project/project-conf-files_services-rabbit.html)
+### `disk`
+
+The `disk` value specifies the size of the persistent disk storage (in MB) to allocate to the service. Services that use persistent storage, such as MySQL, must provide a disk value. Services that use memory instead of persistent storage, such as Redis, do not require a disk value.
+
+```yaml
+mysql:
+    type: mysql:10.2
+    disk: 2048
+```
+
+The current default storage amount per project is 5GB, or 5120MB. You can distribute this amount between your application and each of its services.
+
+## Service relationships
 
-### `disk` {#cloud-yaml-services-disk}
+{{site.data.var.ece}} uses the [`$MAGENTO_CLOUD_RELATIONSHIPS`]({{page.baseurl}}/cloud/env/environment-vars_cloud.html) environment variable to retrieve environment-related relationships. For services to be available to an application in your project, you must specify [*relationships*]({{ page.baseurl }}/cloud/project/project-conf-files_magento-app.html#relationships) between applications and services in the `.magento.app.yaml` file.
```

Comment:
Verify relationships seems a bit vague -- what am I verifying?

That the updates I made in service.yaml and .magento.app.yaml were applied to the configuration? 
Verifying that all my service relationships are defined correctly?
Verifying that the connections work?  (In the older version, the pretty_print command was introduced by "To print the connection information for Elasticsearch)

Reply:
Adding the additional verbiage introducing the php command, and the extra step at the end of the "verify in remote environment" procedure clarifies nicely :-)